spark/src/main/scala/org/apache/mahout/drivers/TestNBDriver.scala

```diff
@@ -76,7 +77,7 @@ object TestNBDriver extends MahoutSparkDriver {
   /** Read the test set from inputPath/part-x-00000 sequence file of form <Text,VectorWritable> */
   private def readTestSet: DrmLike[_] = {
     val inputPath = parser.opts("input").asInstanceOf[String]
-    val trainingSet = drm.drmDfsRead(inputPath)
+    val trainingSet = drm.drmDfsRead(inputPath).par(auto = true)
```

Comment:
@dlyubimov, @pferrel  -does adding `.par(auto = true)` here make sense?  Otherwise there are no explicit partitioning instructions for the input data.